pkg/nodemanager/nodemanager.go

```diff
@@ -72,32 +72,7 @@ var labelReconcileInfo = []struct {
 	primaryKey            string
 	secondaryKey          string
 	ensureSecondaryExists bool
-}{
-	{
-		// Reconcile the beta and the GA zone label using the beta label as
-		// the source of truth
-		// TODO: switch the primary key to GA labels in v1.21
-		primaryKey:            v1.LabelZoneFailureDomain,
-		secondaryKey:          v1.LabelZoneFailureDomainStable,
-		ensureSecondaryExists: true,
```

Comment:
why all stable labels are removed?

Reply:
https://cs.github.com/kubernetes/api/blob/4e8dc44b5ed42b009e0c073c0e4a8f64d4b310ac/core/v1/well_known_labels.go#L35
It seems here all of these Stable types are also deprecated at least thats my understanding correct me if i am wrong.

Reply:
The  labelZoneFailureDomain and LabelZoneFailureDomainStable seem to deprecated..

Labels I removed that have deprecation announced in k8s code and release notes.
```go
		v1.LabelZoneRegion,
	v1.LabelZoneRegionStable,
	 v1.LabelZoneFailureDomain,
	v1.LabelZoneFailureDomainStable,
```
These are the zone based deprecation notices
```go
	// Retained for compat when vendored.  Do not use these consts in new code.
	LabelZoneFailureDomain       = LabelFailureDomainBetaZone   // deprecated
	LabelZoneRegion              = LabelFailureDomainBetaRegion // deprecated
	LabelZoneFailureDomainStable = LabelTopologyZone            // deprecated
	LabelZoneRegionStable        = LabelTopologyRegion          // deprecated
	
```

And for the instance types they are also announced as deprecated

```
		v1.LabelInstanceType,
	v1.LabelInstanceTypeStable
```
Have values of
```go
	LabelInstanceType       = "beta.kubernetes.io/instance-type"
	LabelInstanceTypeStable = "node.kubernetes.io/instance-type"
```
And their deprecation is announced in [here](https://learn.microsoft.com/en-us/azure/aks/use-labels#deprecated-labels)

Reply:
The following three should be kept:

* LabelTopologyZone   = "topology.kubernetes.io/zone"
* LabelTopologyRegion = "topology.kubernetes.io/region"
* LabelInstanceTypeStable = "node.kubernetes.io/instance-type"

Reply:
I added back these three labels 
```go
 map[node.kubernetes.io/instance-type:Standard_D2_v3 topology.kubernetes.io/region:eastus topology.kubernetes.io/zone:eastus-1]
```
They show up in the unit-tests and all.